### DISCUSSION TITLE
Morphologist T1 BIAS Correction Error

### DISCUSSION THREAD

**Discussion by sarafv** [brainvisa/brainvisa.github.io#107] [General]
Hi I have a problem with T1 Bias correction

My T1 is a uni mp2rage image modified (the background noise has beed erased).

The output is :
> 
> ------------------------------ ! VIP Warning: empty volume in VipComputeRobustStatInMaskVolume ------------------------------ 
> ------------------------------ ! VIP Warning: Nothing is left in VipHysteresisThresholding ------------------------------ 
> ------------------------------ !! VIP Error: Empty image ------------------------------ ------------------------------ !! VIP Error: Execution aborted in function 'VipComputeBiasFieldMultiGrid'. ------------------------------ 
> 
> ￼ in T1 Bias Correction 1 
> RuntimeError: System command exited with non null value : 111

Here a copy of metadata :
![image](https://user-images.githubusercontent.com/26764989/168249872-f9b721f1-7449-4b8f-b3aa-98e9f1727db9.png)

Any idea?

Thanks!
Sara

**Comment by denisri** [comment]
Hi,
Well I don't remember exactly but there used to be a problem with images where the background noise has been removed, because the correction algorithm was estimating the noise there and failed when there was none. I don't remember if this has been changed (I know Clara has been looking at it), but can you try again with the image _with noise_ to see if it makes a difference ? If you don't have the original image, you can add arbitrary noise using `AimsAddNoise2Data`, or numpy, or anything else.
Denis
